internal/services/securitycenter/security_center_subscription_pricing_resource_test.go

```diff
@@ -98,6 +98,24 @@ func TestAccSecurityCenterSubscriptionPricing_storageAccountSubplan(t *testing.T
 	})
 }
 
+func TestAccSecurityCenterSubscriptionPricing_cloudPostureExtension(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_security_center_subscription_pricing", "test")
+	r := SecurityCenterSubscriptionPricingResource{}
+
+	data.ResourceSequentialTestSkipCheckDestroyed(t, []acceptance.TestStep{
+		{
+			Config: r.cloudPostureExtension(),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("tier").HasValue("Standard"),
+				check.That(data.ResourceName).Key("resource_type").HasValue("CloudPosture"),
+				check.That(data.ResourceName).Key("extension").IsNotEmpty(),
+			),
+		},
+		data.ImportStep(),
+	})
```

Comment:
could we add another step to the test where we update the extensions, just to confirm that the update functionality works as expected too?

Reply:
Added a look for some extension name

Reply:
In order to test this properly, can we also add a couple of extra test stages to this, similar to above (applying the config and then running an `import`, which explicitly disable the extensions and then enable them again - to ensure that we're testing the update codepath?

Reply:
I have done some manual tests since I am not able to run the tests through the framework it self, we can skip it for now but it was tested